prog/vnet/update_load_balancer_node.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Prog::Vnet::UpdateLoadBalancer < Prog::Base
```

Comment:
Just need to be sure we don't have any existing strand for this prog before deployment